hw/ip/csrng/doc/_index.md

```diff
@@ -191,19 +193,22 @@ It holds the following values:
   <tr>
     <td>31:0</td>
     <td>Reseed Counter</td>
-    <td> Value required by NIST to be held in the state instance.
+    <td> Value required by NIST to be held in the state instance, "indicates the number of requests for pseudorandom
+bits since instantiation or reseeding."
     </td>
   </tr>
   <tr>
     <td>159:32</td>
     <td>V</td>
-    <td> Value required by NIST to be held in the state instance, and is of size <tt>BlockLen</tt>.
+    <td> Value required by NIST to be held in the state instance, and is of size <tt>BlkLen</tt>. It "is updated each
+time another blocklen bits of output are produced."
```

Comment:
why is the last sentence quoted that way that it is? Are they needed at all?

Reply:
I put quotes around it because I copied it verbatim from the NIST specification. I can reword it if you think it's better without the quotes.

Reply:
I've reworded them because I agree with you that the quotes in the current form are confusing.